WordPress/Classes/Models/Post+CoreDataProperties.swift

```diff
@@ -4,13 +4,16 @@ import CoreData
 extension Post {
 
     @NSManaged var commentCount: NSNumber?
+    @NSManaged var disabledPublicizeConnections: [NSNumber: [String: String]]?
```

Comment:
I keep forgetting about all the Core Data subtleties 🤦‍♂️ 
We could make this non-optional in the data model, and non-optional in the code, but `@NSManaged` makes it kind of a lie. Since you don't `init` managed objects manually, core data's "optional" is only enforced on validate/save. So while you can be certain that an object fetched from storage will have a non-optional attribute set, you can't have the same guarantee about a new unsaved object, making it less safe. So I think it's probably best to leave it as optional? 🤷‍♂️